### DISCUSSION TITLE
Is there a How to for Serenity Core Reporting Using Telerik Report Html5Viewer ?

### DISCUSSION THREAD

**Discussion by AhmedAbdellah93** [serenity-is/Serenity#5894] [Help]
i need  a sample for telerik report in serenity core

**Comment by edwardch** [comment]
If you search "telerik reporting" you'll find the wiki article explaining how to add it. [See here](https://github.com/serenity-is/Serenity/wiki/Reporting-Integrating-Telerik-Reporting)

**Reply by edwardch** [reply]
@AhmedAbdellah93 if this is your answer please mark it as answered. It really helps others in future when they face similar issues to yours. 
Thanks!
